lms/djangoapps/course_home_api/outline/v1/serializers.py

```diff
@@ -21,8 +21,35 @@ def get_url(self, tool):
         return request.build_absolute_uri(url)
 
 
+class CourseBlockSerializer(serializers.Serializer):
+    """
+    Serializer for Course Block Objects
+    """
+    blocks = serializers.SerializerMethodField()
+
+    def get_blocks(self, blocks):
+        data = {}
+
+        for block_key in blocks:
+            block_data = {
+                'id': str(block_key),
+                'type': str(block_key.category),
+                'display_name': blocks.get_xblock_field(block_key, 'display_name', str(block_key.category)),
+                'lms_web_url': reverse(
+                    'jump_to',
+                    kwargs={'course_id': str(block_key.course_key), 'location': str(block_key)},
+                    request=self.context['request'],
+                ),
+                'children': [str(child) for child in blocks.get_children(block_key)],
```

Comment:
Nit: `child` -> `child_key`. I think child can have a bit more of an implication that it is a child object, but in this case, they are just key references to the children.